mParticle-Kochava.podspec

```diff
@@ -16,7 +16,7 @@ Pod::Spec.new do |s|
 
     s.ios.deployment_target = "10.3"
     s.ios.source_files      = 'mParticle-Kochava/*.{h,m,mm}'
-    s.ios.dependency 'mParticle-Apple-SDK/mParticle', '~> 8.0'
+    s.ios.dependency 'mParticle-Apple-SDK/mParticle', '>= 8.2'
```

Comment:
Wouldn't think this would need to be changed... we definitely can't use `>=` syntax because it would prevent us from ever making backwards incompatible changes to the API between kits and core without breaking existing kits

Reply:
@peterjenkins If I use '~> 8.2'  does that mean if they come out with an 8.3 it won't automatically get pulled in? I believe thats why I made this change (well that and Sam's CR request on this)

Reply:
Nevermind I checked myself